content/en/introduction.org

```diff
@@ -10,12 +10,12 @@
 
 Hewwo hewwo nyaoo nyaoo~~
 
-My name is Yayoi Shizumi (静海やよい in Japanese) and I am a magical fairy cat princess from a mystical realm near the North West Passage! I was sent to this realm to protect it. Unfortunately, my travel here was far from uneventful and I am no longer able to speak. I have had to find new ways to communicate in order to protect those I was sent to protect, my kittens!
+My name is Yayoi Shizumi (静海やよい in Japanese) and I am a magical fairy cat princess from a mystical realm near the North West Passage! I was sent to this realm to protect it. Unfortunately, my travel here was far from uneventful and I am no longer able to speak. I have had to find new ways to communicate in order to protect those I was sent +to protect+ (here for, "protect" seems redundant here), my kittens!
 
 Aside from using my powers to protect my kittens I've also been charged with the task of spreading positivity and love! I do this using the very networks evil forces are trying to use to spread hate and negativity! On these networks, I play a variety of games and chat with anyone who joins, kitten or not. So please check me out! I live stream on [[https://www.twitch.tv/yayoi_chi][Twitch]] and post videos on [[https://www.youtube.com/@yayoi-chi][YouTube]]!
 
 You can get recaps, and updates on this website, which the kind folks taking care of me in your world have made for me, but you can also get updates on [[https://mastodon.world/@yaya][Mastodon]] and my [[https://www.youtube.com/@yayoi-chi/community][YouTube Community Tab]]! I love helping and have also created some [[{{% ref guides %}}][Guides]] and [[{{% ref reviews %}}][Reviews]] of some of my favorite games for you!
 
 My mission above all is to be a great friend! So please help me out and connect with me. I might not be able to talk anymore, but I can still communicate and I love getting to know new people. Also, if you would like to become one of my kittens, I'd love to have you. Just follow me on Twitch or subscribe on YouTube!
 
-Thank you so so so much for listening! I hope I will be able to get to know you!
+Thank you so so so much for +listening+ (reading?)! I hope I will be able to get to know you!
```

Comment:
You commit this one has is. I should probably not use words that refer to spoken language 😅